packages/contracts-bridge/contracts/BridgeRouter.sol

```diff
@@ -155,6 +158,39 @@ contract BridgeRouter is Version0, Router {
         emit Send(_token, msg.sender, _destination, _recipient, _amount, false);
     }
 
+    /**
+     * @notice Send tokens to a hook on the remote chain
+     * @param _token The token address
+     * @param _amount The token amount
+     * @param _destination The destination domain
+     * @param _hook The hook contract on the remote chain
+     * @param _extraData Extra data that will be passed to the hook for
+     *        execution
+     */
+    function xsend(
```

Comment:
we should probably rename this function? `sendToHook` ?

Reply:
another naming note: not clearest naming to me that `hook` connotes a contract which implements the logic for a hook